experimental/unitsconversion/Cargo.toml

```diff
@@ -25,9 +25,7 @@ displaydoc = { version = "0.2.3", default-features = false }
 icu_locid = { workspace = true }
 icu_provider = { workspace = true, features = ["macros"] }
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
-tinystr = { workspace = true, features = ["zerovec"], default-features = false }
-zerovec = { workspace = true, features = ["yoke"] }
-
+zerovec = { workspace = true, features = ["yoke", "serde"] }
```

Comment:
Issue: Don't require "serde" here because it gets enabled by the "serde" feature below.

Reply:
done